modules/webrtc/doc_classes/WebRTCPeerConnection.xml

```diff
@@ -5,7 +5,7 @@
 	</brief_description>
 	<description>
 		A WebRTC connection between the local computer and a remote peer. Provides an interface to connect, maintain and monitor the connection.
-		Setting up a WebRTC connection between two peers from now on) may not seem a trivial task, but it can be broken down into 3 main steps:
+		Setting up a WebRTC connection between two peers from now on may not seem a trivial task, but it can be broken down into 3 main steps:
```

Comment:
I'd suggest removing the `from now on`, I'm not sure it adds much and actually confuses me a bit. "From now on" refers to a past change, what change are we talking about?

Reply:
Done and done.